lightning/src/chain/keysinterface.rs

```diff
@@ -77,8 +77,8 @@ pub trait KeysInterface: Send + Sync {
 	/// Get a new set of ChannelKeys for per-channel secrets. These MUST be unique even if you
 	/// restarted with some stale data!
 	fn get_channel_keys(&self, inbound: bool) -> ChannelKeys;
-	/// Get a secret for construting an onion packet
-	fn get_session_key(&self) -> SecretKey;
+	/// Get a secret and PRNG seed for construting an onion packet
```

Comment:
Maybe not divergence on the spec, we are outputing a PRNG not a CSPRNG

Reply:
ChaCha is a CSPRNG.